sql/core/src/main/scala/org/apache/spark/status/api/v1/sql/ApiSqlRootResource.scala

```diff
@@ -31,4 +31,15 @@ private[v1] class ApiSqlRootResource extends ApiRequestContext {
   def sqlList(
       @PathParam("appId") appId: String,
       @PathParam("attemptId") attemptId: String): Class[SqlResource] = classOf[SqlResource]
+
+  @Path("applications/{appId}/diagnostics")
```

Comment:
If you want to add this in this PR, `REST API` should be documented here.
- https://spark.apache.org/docs/latest/monitoring.html#rest-api

Reply:
resolved as duplication